evennia/contrib/actions/__init__.py

```diff
@@ -0,0 +1,58 @@
+from evennia import ObjectDB, DefaultCharacter, DefaultRoom, CmdSet
+from evennia.contrib.actions.actions import Action
+from evennia.contrib.actions.handlers import CharacterActionHandler, RoomActionHandler
+from evennia.contrib.actions.scripts import ActionSystemScript
+from evennia.contrib.actions.typeclasses import ActionCharacter, ActionRoom, ActionExit
+from evennia.contrib.actions.commands import (CmdAlarm, CmdActSettings, CmdActions,
+    CmdStop, CmdDone, CmdQueue)
+from evennia.contrib.actions.debug import ActionDebugCmdSet
+from evennia.contrib.actions.setup import setup
+
+class ActionCmdSet(CmdSet):
```

Comment:
The base `ActionCmdSet` seems to belong and be easier to find in the `command.py` module then imported here rather than defined here. There is (in principle, at least) no need to import the individual commands here since people can just import the cmdset and add it to their default cmdset in one go (but you may want to have the commands available for people to pick-and-choose from I suppose).